libflux/src/core/ast/flatbuffers/.gitattributes

```diff
@@ -0,0 +1 @@
+ast_generated.rs binary
```

Comment:
Does marking these files as binary make it so that git doesn't show their diff?

Reply:
Yeah, `git diff` becomes very hard to navigate with those generated files. For our purposes, they might as well be binary files, so I marked them as such.